include/sprite.h

```diff
@@ -56,10 +76,12 @@ typedef struct {
     // Flags
     // 0x200 = something about updating animations (sub_8003638)
     u16 unk2E;
-    u16 unk30;
-    u16 unk32;
-    u16 unk34;
-    u16 unk36;
+
+    // apparently NOT signed?
```

Comment:
Maybe we don't have any instances which prove they are not signed?

Reply:
I don't know. I'll leave the comment there for now